generic/ras_extended.py

```diff
@@ -173,8 +173,23 @@ def test1_vpdupdate(self):
                                         "head -1", shell=True).strip()
         if path_db:
             copyfile_path = os.path.join(self.outputdir, 'vpd.db')
-            copyfile(path_db, copyfile_path)
+            shutil.copyfile(path_db, copyfile_path)
             self.run_cmd("vpdupdate --path=%s" % copyfile_path)
+        if os.path.exists('/var/lib/lsvpd/run.vpdupdate'):
+            path = '/var/lib/lsvpd/run.vpdupdate'
+        elif os.path.exists('/var/lib/lsvpd/run.vpdupdate'):
```

Comment:
I don't see any difference between `if` condition file path and `elif` condition file path. Both are `/var/lib/lsvpd/run.vpdupdate`.